### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by sarthakkhandelwal7**

### CODE REVIEW

File: binary_decimal_converter.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+class BinaryDecimalConverter:
+    def __init_(self):
```
Comment (nikhilkhandelwal) [inline binary_decimal_converter.py]: it is two  __init__ not  __init_

Comment (sarthakkhandelwal7) [reply binary_decimal_converter.py]: ok

File: binary_decimal_converter.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    print("To convert decimal to binary press 1")
+    print("To convert binary to decimal press 2")
+    try:
+        choice = int(input("Enter your choice:"))
+    except:
+        print("Enter correct choice")
+        ask = input("Want to converter again y/n:")
+        continue
+    if choice==1:
+        calculator.gets_power_of_two()
```
Comment (nikhilkhandelwal) [inline binary_decimal_converter.py]: what is get power of two? have you even ran it and checked that it works for a few examples?

Comment (sarthakkhandelwal7) [reply binary_decimal_converter.py]: Code is correct. I have checked it for about 10-20 examples. Please review the latest commit.

Comment (sarthakkhandelwal7) [reply binary_decimal_converter.py]: code

**Pull request merged**